package.json

```
@@ -2,6 +2,7 @@
   "name": "starlight",
   "type": "module",
   "version": "0.0.1",
+  "nodeVersion": "^18.0.0",
```

```suggestion
  "engines": {
    "node": ">=18"
  },
```
Let's follow the [official docs](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#engines) on this.